From d88e20497228ca381a7cae24ccce55611c60c0d4 Mon Sep 17 00:00:00 2001
From: Kas User <kas@example.com>
Date: Thu, 31 Aug 2023 08:48:55 +0000
Subject: [PATCH] enable hotreload and use yaml conf file

---
 init/systemd.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/init/systemd.in b/init/systemd.in
index 8838bc5ce..c763babbe 100644
--- a/init/systemd.in
+++ b/init/systemd.in
@@ -8,7 +8,7 @@ After=network.target
 Type=simple
 EnvironmentFile=-/etc/sysconfig/@FLB_OUT_NAME@
 EnvironmentFile=-/etc/default/@FLB_OUT_NAME@
-ExecStart=@CMAKE_INSTALL_FULL_BINDIR@/@FLB_OUT_NAME@ -c /@CMAKE_INSTALL_SYSCONFDIR@/@FLB_OUT_NAME@/@FLB_OUT_NAME@.conf
+ExecStart=@CMAKE_INSTALL_FULL_BINDIR@/@FLB_OUT_NAME@ -Y -c /@CMAKE_INSTALL_SYSCONFDIR@/@FLB_OUT_NAME@/@FLB_OUT_NAME@.yaml
 Restart=always
 
 [Install]
